Microsoft.Toolkit.Services/Microsoft.Toolkit.Services.csproj

```diff
@@ -1,7 +1,7 @@
 ﻿<Project Sdk="MSBuild.Sdk.Extras">
 
   <PropertyGroup>
-    <TargetFrameworks>uap10.0.16299;netstandard2.0;net461</TargetFrameworks>
+    <TargetFrameworks>uap10.0.16299;netstandard2.0;net462</TargetFrameworks>
```

Comment:
Should we update docs? Changing net framework version. Not sure, but please verify. @nmetulev ?